src/components/application_manager/test/CMakeLists.txt

```diff
@@ -129,12 +141,29 @@ set(LIBRARIES
   jsoncpp
 )
 
-create_test(am_commands_test "${COMMANDS_SOURCES}" "${LIBRARIES}")
+add_library(mock_message_helper_library ${MOCK_MESSAGE_HELPER_SOURCE})
+target_link_libraries(mock_message_helper_library ${LIBRARIES})
```

Comment:
@anosach-luxoft, as I understood cmake documentation, this parameter have to be in my case, moreover without it some test not buildable. So, sorry, I prefer stay stay it "as is". Thank you for your notice.